simple_template_server/README.md

```diff
@@ -0,0 +1,50 @@
+Template Development Server
+===================================
+
+This is a simple Snippet Server for development of Snippet Templates. It ignores
+the client configuration including locale, Firefox version and release channels
+and just returns all the snippets to the requesting client. If you only care for
+Snippet Template development this is the server for you.
+
+Installation
+------------
+
+1. Create and activate a Python virtualenv
+
+    ``` shell
+    virtualenv venv
+    . venv/bin/activate
+    ```
+
+2. Install the needed packages
+
+    ``` shell
+    pip install -r requirements.txt
```

Comment:
I initially got an error and a separate warning "You are using pip version 6.0.8, however version 8.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command."

Fixing the warning also fixed the error.

Now I get another error, here is the output:

```
(venv)[scott@localhost snippets-service]$ pip install -r requirements.txt
Collecting hashin==0.4.1 (from -r requirements.txt (line 1))
  Downloading hashin-0.4.1-py2-none-any.whl
Collecting Django==1.8.12 (from -r requirements.txt (line 5))
  Downloading Django-1.8.12-py2.py3-none-any.whl (6.2MB)
    100% |████████████████████████████████| 6.2MB 252kB/s 
Collecting dj-database-url==0.3.0 (from -r requirements.txt (line 8))
  Downloading dj_database_url-0.3.0-py2.py3-none-any.whl
Collecting MySQL-python==1.2.3c1 (from -r requirements.txt (line 11))
  Downloading MySQL-python-1.2.3c1.tar.gz (89kB)
    100% |████████████████████████████████| 92kB 1.8MB/s 
    Complete output from command python setup.py egg_info:
    sh: mysql_config: command not found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-Goup1H/MySQL-python/setup.py", line 15, in <module>
        metadata, options = get_config()
      File "setup_posix.py", line 43, in get_config
        libs = mysql_config("libs_r")
      File "setup_posix.py", line 24, in mysql_config
        raise EnvironmentError("%s not found" % (mysql_config.path,))
    EnvironmentError: mysql_config not found
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-Goup1H/MySQL-python/
```

It is my understanding that mysql should be packaged and setup inside virtualenv and I shouldn't need it installed globally on my system, correct?